### ISSUE TITLE
Dont detect some errors!

### ISSUE THREAD

**Issue by hellresistor** [azohra/shell-linter#24]
Hi!
I am trying your Action from here [MarketPlace](https://github.com/marketplace/actions/shell-linter?version=v0.3.0
)

my .yml file:
```
name: LintCheck
on:
  
  push:
    branches:
      - master
  pull_request:
    
jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Lint check
        uses: azohra/shell-linter@v0.3.0
```

I have been detected this things:
Correct command: `echo -e "${bkwhite}\\n\\n                         ${bld}${red}%@. \\n                      #@@@@.\\n${bkwhite}"
`
I have set colors before, are OK.

Tests:
 - `echo -e ${bkwhite}\n\n                         ${bld}${red}%@. \\n                      #@@@@.\\n${bkwhite}"
` . Should detect missing 1st double-quote
 - `eco -e "${bkwhite}\\n\\n                         ${bld}${red}%@. \\n                      #@@@@.\\n${bkwhite}"`. Should Detect mistake on "echo" command or check existing bin.

**Comment by hellresistor** [comment]
https://github.com/koalaman/shellcheck/issues/1936#issuecomment-623168668
